cpp/CMakeLists.txt

```diff
@@ -363,6 +364,7 @@ else()
     target_link_libraries(cugraph
         PUBLIC
             rmm::rmm
+            rt
```

Comment:
Dumb question, is this intention ? What does this do.

Reply:
same as other one